node_lambnik/src/tiler/src/tile-interface.js

```diff
@@ -35,12 +35,7 @@ export const home = () => {
 }
 
 // img endpoint to verify binary content serving
-export const img = () => new Promise((resolve, reject) => {
-    fs.readFile(path.join(__dirname, 'res/img.png'), (err, data) => {
-        if (err) reject(err)
-        else resolve(data)
-    })
-})
+export const img = () => readFile(path.join(__dirname, 'res/img.png'))
```

Comment:
I see this is dropped in #40, but since the image and its directory are removed in this PR, it would be a little cleaner to get rid of this in this one, too.